src/main/cpp/subsystems/HatchIntake.cpp

```diff
@@ -6,11 +6,13 @@
 /*----------------------------------------------------------------------------*/
 
 #include "subsystems/HatchIntake.h"
+
 #include "commands/HatchIntakeDefaultCommand.h"
 #include "commands/HatchIntakeSpeedDefaultCommand.h"
-#include "utilities/LineCalculator.h"
 #include "commands/PivotHatch.h"
 
+#include "utilities/LineCalculator.h"
```

Comment:
Why is `LineCalculator` separated by a whitespace?

All of the include files can be together. The header for the cpp can be separated by whitespace.